src/Illuminate/Cache/MemoizedStore.php

```diff
@@ -108,6 +108,7 @@ public function put($key, $value, $seconds)
     /**
      * Store multiple items in the cache for a given number of seconds.
      *
+     * @param  array  $values
```

Comment:
Since `prefix()` expects a `string`, so this can be reflected here:
```suggestion
     * @param  array<string, mixed>  $values
```